src/pages/Home.vue

```diff
@@ -113,17 +131,47 @@ export default {
   },
   methods: {
     ...mapActions("plioItems", ["purgeAllPlios"]),
+    ...mapActions("sync", ["startLoading", "stopLoading"]),
+    async resetSearchString() {
+      // reset the search string to ""
+      // fetch all the plios again
+      if (this.searchString != "") {
+        this.startLoading();
+        await this.fetchAllPlioIds();
+        this.stopLoading();
+        this.searchString = "";
+      }
+    },
     async fetchAllPlioIds() {
       var uuidOnly = true;
       await PlioAPIService.getAllPlios(uuidOnly).then((response) => {
         // if no plios exist, show the warning else hide the warning
-        if (response.data.length <= 0) this.hasAnyPlios = false;
-        else this.hasAnyPlios = true;
-
-        this.prepareTableData(response.data);
+        if (response.data.count <= 0) {
+          // if not plios exist, show the warning
+          this.hasAnyPlios = false;
+        } else {
+          // if plios exist
+          this.hasAnyPlios = true; // hide the warning and show the table
+          this.totaNumberOfPlios = response.data.count; // set total number of plios and show the paginator
+          this.numberOfPliosPerPage = response.data.page_size; // set the page size
```

Comment:
From what I understand this only gives the number of plios in the current fetched page right?
```suggestion
          this.numberOfPliosOnPage = response.data.page_size; // set the page size
```